### PR TITLE
Replace Firebase with AuthJS

### PR DESCRIPTION
**Pull request by rhit-windsors**

### CODE REVIEW

**Comment by rhit-windsors** [comment]
Finally ready I swear

File: kubernetes_init/frontend/deployment-frontend.yaml [hunk 1]
```diff
@@ -16,46 +16,75 @@ spec:
...
-            secretKeyRef:
-              name: secret-env
-              key: MONGODB_USERNAME
-        - name: MONGODB_PASSWORD
-          valueFrom:
-            secretKeyRef:
-              name: secret-env
-              key: MONGODB_PASSWORD
-        
+        - name: container-test-frontend
```
Comment (bveal52) [inline kubernetes_init/frontend/deployment-frontend.yaml:19]: container-test-frontend?

Comment (bveal52) [reply kubernetes_init/frontend/deployment-frontend.yaml:19]: can we rename it

**Review by bveal52 — CHANGES_REQUESTED** [review]
look at comments

File: apps/frontend/app/components/auth/SignInModal.tsx [hunk 2]
```diff
@@ -22,29 +22,29 @@ export const SignInModal = ({ afterSignIn }: SignInModalProps) => {
...
-							setButtonDisabled(false);
-						});
+						// await authService.signInWithEmailAndPassword(
+						// 	email,
+						// 	password
+						// ).then(() => {
+						// 	afterSignIn();
+						// }).catch((error) => {
+						// 	alert(`Problem logging in: ${error}`);
+						// 	setButtonText(DEFAULT_SIGN_IN_TEXT);
```
[the remark this answers is not in the document]
Comment (rhit-windsors) [reply apps/frontend/app/components/auth/SignInModal.tsx:45]: This page will most likely need to be redone/deleted in a future version.

File: kubernetes_init/frontend/deployment-frontend.yaml (hunk 1 above)
Comment (rhit-windsors) [reply kubernetes_init/frontend/deployment-frontend.yaml]: renamed this and the backend

**Review by bveal52 — APPROVED** [review]
looks good

**Pull request merged**